test/test_masamune.rb

```diff
@@ -81,19 +79,14 @@ def test_chained_method_calls
 
     msmn = Masamune::AbstractSyntaxTree.new(blocks)
     methods = msmn.all_methods
-    assert methods.size == 8
+    assert_equal methods.size, 8
 
-    method_names = methods.map {|m| m[:token]}
+    method_names = methods.map {|m| m.token}
```

Comment:
Might as well, while we're here 😄

```suggestion
    method_names = methods.map(&:token)
```

Reply:
Updated in https://github.com/gazayas/masamune-ast/pull/60/commits/f75a4f96bc16a84513f3907274165034927ed8fc